test/page-objects/submit_question.js

```diff
@@ -0,0 +1,19 @@
+const { question } = require('paths');
+const BasePage = require('test/page-objects/base');
+
+class SubmitQuestionPage extends BasePage {
+  constructor(page, hearingId, questionId) {
+    super(page);
+    this.page = page;
```

Comment:
I think this is done in super, so it can be removed.

Reply:
Cool, have cleaned up